Lecture4/App.js

```diff
@@ -1,14 +1,41 @@
 import React from "react";
+import { useState } from "react"
```

Comment:
You can do this-
import React, {useState} from 'react';
Also, a blank line after global import will look more readable.